src/mlpack/bindings/julia/julia_util.cpp

```diff
@@ -485,7 +493,7 @@ size_t* CLI_GetParamURow(const char* paramName)
  */
 size_t CLI_GetParamMatWithInfoRows(const char* paramName)
 {
-  return std::get<1>(CLI::GetParam<std::tuple<data::DatasetInfo, arma::mat>>(
+  return std::get<1>( CLI::GetParam<std::tuple<data::DatasetInfo, arma::mat>>(
```

Comment:
What is the reason of the added space here, to improve readability?

Reply:
Nice catch, the reason is an incorrect applied vim macro. :)  Let me find all errors like this and fix them. :+1: